playground/mocks/data/idp/idx/terminal-return-email-consent-denied.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
SIW gets this remediation when the User clicks on the `No, it's not me` button from the `consent` screen.